src/types/sidebar-link.ts

```diff
@@ -1,6 +1,9 @@
 export interface SidebarLink {
   to?: string
-  title: string
+  title?: string
+  key?: string
   children?: SidebarLink[]
   defaultOpened?: boolean
+  component?: React.FC
+  onClick?: () => void
```

Comment:
Allows "New Question" and "New Dashboard" to reset its state when clicked on in the sidebar. Also allows the "From Scratch / From Template" popover menu to show on the sidebar.